New version: getfloma.floma version 1.1.8

AdityaT248 opened the pull request:
This PR adds version 1.0.4 of floma.

**New features in v1.0.4:**
- Template system with 11 pre-built workflow templates (paid feature)
- Preview/dry-run mode improvements (now actually prevents execution)
- Workflow-specific daemon scheduling
- Better UX and error handling

**Package Information:**
- Publisher: GetFloma
- Package URL: https://getfloma.com
- License: MIT
- Binary hosted on Cloudflare R2

**Verification:**
- SHA256 verified
- Binary URL accessible
- All manifest files validated
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/309618)

AdityaT248 committed:
New version: getfloma.floma version 1.0.4

AdityaT248 commented:
Hi @stephengillie,

The installer URLs point to our CloudFlare R2 CDN which hosts our
official releases.

Evidence of ownership:

Publisher domain: https://getfloma.com/ (owned by us)
Download page: https://getfloma.com/ shows installation instructions
R2 bucket: pub-3da12ce477ba4b969191f6514d7505ff.r2.dev is our
CloudFlare R2 CDN
The R2 URLs are the official download source referenced on our website.
We can provide additional verification if needed.

Thank you!

stephengillie commented:
Hi @AdityaT248,

Manual Validation ended with:
<img width="812" height="436" alt="image" src="https://github.com/user-attachments/assets/814c9b51-928a-4034-96a4-7cc33660d062" />
The application seems to be looking for a file that isn't in the filesystem. Is this expected?

> The installer URLs point to our CloudFlare R2 CDN which hosts our
official releases.

Our test is to navigate from the `PackageUrl` to the `InstallerUrl`. Unfortunately, the package appears to be paywalled:
<img width="798" height="609" alt="image" src="https://github.com/user-attachments/assets/bbfb8ee9-bd20-4ac5-82e7-83e2b2b44939" />
Clicking the "Start Free" button in the left column didn't seem to have any effect, at least in my browser. Is there another public page that has a link to the `InstallerUrl`?

stephengillie commented:
Automatic Validation ended with: 
> No logs after 10 retries.

(Automated response - build 1140.)

AdityaT248 committed:
Update getfloma.floma to 1.1.8 - Fix Windows binary distribution

Changes in v1.1.8:
- Changed Windows distribution from single-file exe to zip package
- Resolves PyInstaller temp extraction error (python311.dll issue)
- Uses onedir mode to avoid Windows security/antivirus blocking
- InstallerType changed from 'portable' to 'zip' with nested portable installer
- Updated SHA256 hash for new zip file
- Binary now extracts to floma folder with floma.exe inside

This addresses the validation issue where the previous .exe failed to run
due to temp directory extraction being blocked by Windows security.

stephengillie commented:
Automatic Validation ended with: 
> 
> Installation failed with exit code -1978335141
> 2025-11-04 23:02:06.719 [CLI ] Unable to locate nested installer at: C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\getfloma.floma.1.1.8\extracted\floma\floma.exe 
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
  | 8A15005B | -1978335141 | 2316632155 | APPINSTALLER_CLI_ERROR_NESTEDINSTALLER_NOT_FOUND | Failed to find the nested installer in the archive. Does the `RelativeFilePath` have the previous version number in it? | 


(Automated response - build 1140.)

stephengillie commented:
> @stephengillie,
> 
> If you click on the alternative install methods dropdown (under the To update: brew update && brew upgrade floma), on getfloma.com , you will see the InstallerUrls. 

Thanks - I was having trouble differentiating UI elements from styled text. 

> I am currently uploading a fix to the windows download error, do you want me to create a new pr or push the updated release in this PR.

> Version: Updated from 1.0.4 to 1.1.8

A `PackageVersion` change might require updating the file path, so it might be easier to open a new PR. I've added this application to my automation's autowaiver list, so future PRs will clear the `Validation-Domain` label automatically.

stephengillie commented on manifests/g/getfloma/floma/1.1.8/getfloma.floma.installer.yaml:
```diff
@@ -0,0 +1,15 @@
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+  - RelativeFilePath: floma\floma.exe
```
```suggestion
  - RelativeFilePath: floma.exe
```

stephengillie reviewed (changes requested):
Manual Validation ended with:
<img width="781" height="323" alt="image" src="https://github.com/user-attachments/assets/22681731-2565-46d0-9bae-11ff441e76e6" />

It seems that the executable is in the zip archive's root.
